Kickstarter-iOS/AppDelegate.swift

```diff
@@ -30,6 +30,9 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
+    UIView.doBadSwizzleStuff()
+    UIViewController.doBadSwizzleStuff()
```

Comment:
welp here's where we're at. doing our bad swizzle stuff first thing when the app launches.

at least we are being honest now and not hiding deep somewhere else!